site/content/docs/main/code-standards.md

```diff
@@ -8,6 +8,19 @@ toc: "true"
 
 When opening a pull request, please fill out the checklist supplied the template. This will help others properly categorize and review your pull request.
 
+### PR title
+
+Make sure that the pull request title summarizes the change made (and not just "fixes issue #xxxx"):
+
+Example PR titles:
+
+ - "Check for nil when validating foo"
+ - "Issue #1234: Check for nil when validating foo"
```

Comment:
since we have issue placeholder in PR template, I would discourage issue ref in PR title (unless this helps with any automation, like what we have for JIRA in OADP)

But I would also add a section explaining how to use the PR template, I saw a lot of misusage

maybe update it with comments to help users fill it (example https://raw.githubusercontent.com/dora-metrics/pelorus/master/.github/PULL_REQUEST_TEMPLATE/general_template.md)

Reply:
While I don't want to require use of the issue in the title/commit message, I can see some value in including it, since PR template won't show up when looking at commit history. That's why I gave both options -- i.e. if you want the issue in title, that's fine, as long as there's also a description, but just a description is fine too.

Reply:
The most important point (and the main reason for this change) is to make sure that every PR gets a descriptive title -- that "fix issue #foo" alone is insufficient.

Reply:
@sseago unrelated to this PR going in, we can perhaps create github actions to ensure proper naming of PRs?